packages/E2ETest/run_wdio.js

```diff
@@ -69,26 +70,39 @@ function OverrideHyperV() {
   }
 }
 
-OverrideHyperV();
-
 const Launcher = require('@wdio/cli').default;
 
 const wdio = new Launcher('wdio.conf.js', { specs: opts });
 
 function parseLog(logfile) {
   const xmlString = fs.readFileSync(logfile);
-  let name;
+  let failures = {};
   parser.parseString(xmlString, (err, res) => {
     if (!res.testsuites) {
-      name = 'something went wrong';
+      failures = 'something went wrong';
     } else {
-      const attr = res.testsuites.testsuite[0].ATTR;
-      if (attr.errors > 0 || attr.failures > 0) {
-        name = attr.name;
+      for (let i = 0; i < res.testsuites.testsuite.length; i++) {
+        const attr = res.testsuites.testsuite[i].ATTR;
+        if (attr.errors > 0 || attr.failures > 0) {
+          let name = attr.name;
+          failures[name] = {};
+          const testcases = res.testsuites.testsuite[i].testcase;
+          for (let j = 0; j < testcases.length; j++) {
```

Comment:
nit: this could really use some whitespace to make it a bit more readable

Reply:
Also named variables instead of indices. See the other comment about not using raw for loops.

Reply:
the spacing matches what lint expects

Reply:
I mean new lines. Lint doesn't care about those.

Reply:
That doesn't seem to be the case.
![image](https://user-images.githubusercontent.com/22989529/81506438-2c637580-92ab-11ea-91cd-6c75dab4081d.png)
![image](https://user-images.githubusercontent.com/22989529/81506448-3ab19180-92ab-11ea-8afa-8e36204d29a2.png)

Reply:
@asklar Prettier will warn about Windows style line feeds. You'll want to change what your editor is set to likely. It should not complain about single new lines.

Reply:
Actually I think it will also complain about empty lines that start directly after a conditional.

Reply:
What I meant with "lint doesn't care about those" is that it won't add them for you,